pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/ByteSchema.java

```diff
@@ -33,15 +33,15 @@ public static ByteSchema of() {
     }
 
     private static final ByteSchema INSTANCE = new ByteSchema();
-    private static final SchemaInfo SCHEMA_INFO = new SchemaInfo()
+    public static final SchemaInfo SCHEMA_INFO = new SchemaInfo()
         .setName("INT8")
         .setType(SchemaType.INT8)
         .setSchema(new byte[0]);
```

Comment:
Could also modified?
```java
.setSchema(schema.toString().getBytes(UTF_8));
```

Reply:
There is no equivalent schema in avro that represents byte or short. That's why I've kept it the same.

Reply:
I get it, thanks